Makefile

```diff
@@ -365,20 +355,6 @@ docker_test:
 docker_unit_test:
 	go run test.go -flavor $(flavor) unit
 
-# Release a version.
-# This will generate a tar.gz file into the releases folder with the current source
-release: docker_base
```

Comment:
I removed `docker/release` and the corresponding make command as it seems to not be used and very ancient.

Reply:
Agreed. It was being used by the pre-GH release process.